contracts/managers/AddressManager.sol

```
@@ -184,7 +183,6 @@ contract AddressManager is
      * @return success A boolean indicating whether the role was successfully created
      */
     function createNewRole(bytes32 newRole) external onlyOwner returns (bool success) {
-        require(!_protocolRoles.contains(newRole), AddressManager__RoleAlreadyExists());
         success = _protocolRoles.add(newRole);
```

If the role already exists, the `add` will silently return the `false` status, I'd prefer keeping it this way.  If explicit handling of role existence is desired, you can assert the `success` state.